test.sh

```diff
@@ -27,5 +27,7 @@ protoc -I=. --go_out=. github.com/googleinterns/step224-2020/config/proto/*.prot
 # Find finds all of the files ending in _test.go and returns their relative
 # paths. Sed removes the the file part of that relative path.
 # Then, we remove uniques.
-go test $(find ./github.com/googleinterns/step224-2020
- -type f -name '*_test.go' | sed -r 's|/[^/]+$||' -u)
+go test $(find ./github.com/googleinterns/step224-2020 -type f -name '*_test.go' | sed -r 's|/[^/]+$||' -u)
```

Comment:
Alternative thing, that shouldn't be a blocker to merging this:

There must be a better way to deal with this :-P

Reply:
I have added a new version :-P